### ISSUE TITLE
[Bug] Crashed Dyld Message:Symbol not found: _kCGColorSpace  in iOS 9.x

### ISSUE THREAD

**Issue by weacw** [Unity-Technologies/arfoundation-samples#706]
**Describe the bug**
![image](https://user-images.githubusercontent.com/11573490/102763799-2727e680-43b5-11eb-93f1-bc81fdb62526.png)

this problem of `Dyld Message:Symbol not found: _kCGColorSpace` appeared in iOS 9.3.3/9.3.5.
I modified the Unity Xcode code to change its minimum available system version to 9.0.

Our ARFoundation setting:
![image](https://user-images.githubusercontent.com/11573490/102763619-e5973b80-43b4-11eb-8457-dbc7a89fa067.png)

I used the grep command to find the `_kCGColorSpaceLinearGray` string in Libraries/com.unity.xr.arkit/Runtime/iOS/Xcode1200/libUnityARKit.a.

```command
nm UNITY_LIBUNITYARKIT.A_PATH  -arch arm64 | grep '_kCGColorSpaceLinearGray'
```

```command
 grep -r '_kCGColorSpaceLinearGray' .
Binary file ./Libraries/com.unity.xr.arkit/Runtime/iOS/Xcode1200/libUnityARKit.a matches
```

**Smartphone (please complete the following information):**
 - Device: iPhone 6
 - OS: 9.3.3/9.3.5
 - Unity version:2019.4.15f1
 - ARFoundation version:4.1.1

![image](https://user-images.githubusercontent.com/11573490/102763743-124b5300-43b5-11eb-90f2-97687d4d5a07.png)

**Comment by weacw** [comment]
Is there any progress? @tdmowrer

**Comment by tdmowrer** [comment]
IIRC, we don't support anything before iOS 10 (which is already pretty old and has a very limited reach). Why do you need to support iOS 9.x?

**Comment by weacw** [comment]
> IIRC, we don't support anything before iOS 10 (which is already pretty old and has a very limited reach). Why do you need to support iOS 9.x?

Thank you for your reply.

`You are right. ARKit and ARFoundation not supported iOS 10 and iOS 9.x.`
We currently treat AR as a module to enhance product attributes. And our APP need to supported iOS 10 and iOS 9.x. When the user does not support AR, we will hide the entrance or prompt that it is not supported. But now iOS9.x users will crash when opening our APP and cannot use the APP normally.

In other words, the current iOS9.x launches the APP will crash. The reason is that `libUnityARKit.a` contains `_kCGColorSpaceLinearGray`

**Comment by tdmowrer** [comment]
We do support iOS 10 (AR won't be available, but the app will not crash). We have no plans to support iOS 9.

**Comment by weacw** [comment]
> We do support iOS 10 (AR won't be available, but the app will not crash). We have no plans to support iOS 9.

Thank you for your reply.
The reason for the current crash should be that `libUnityARKit.a` is loaded when the APP starts, and the crash is caused by iOS 9.x not supporting [_kCGColorSpaceLinearGray](https://developer.apple.com/documentation/coregraphics/kcgcolorspacelineargray). Is there any solution to avoid this problem?

Currently this application is serving an international bank. Their system is relatively old, so they need to support iOS9.x

**Comment by tdmowrer** [comment]
Yep, I understand the issue; however, we have no plans to support iOS 9, and there is no workaround to avoid this issue on devices running iOS 9.

**Comment by weacw** [comment]
> Yep, I understand the issue; however, we have no plans to support iOS 9, and there is no workaround to avoid this issue on devices running iOS 9.
Is the ‘libUnityARKit.a’ open source?
can modified it to support iOS 9.x by myself?

**Comment by tdmowrer** [comment]
No, it is not open source. There is no way to support iOS 9 with ARFoundation.
